main.go

```diff
@@ -0,0 +1,41 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/DataDog/test-infra-definitions/registry"
+	_ "github.com/DataDog/test-infra-definitions/registry/scenarios"
+
+	"github.com/pulumi/pulumi/sdk/v3/go/pulumi"
+	"github.com/pulumi/pulumi/sdk/v3/go/pulumi/config"
+)
+
+const (
+	scenarioEnvVarName = "PULUMI_SCENARIO"
+	scenarioParamName  = "scenario"
+
+	dummyScenario = "dummy"
+)
+
+func main() {
+	pulumi.Run(func(ctx *pulumi.Context) error {
+		scenarioName := os.Getenv(scenarioEnvVarName)
+		rootConfig := config.New(ctx, "")
+		if s := rootConfig.Get(scenarioParamName); s != "" {
+			scenarioName = s
+		}
+
+		if scenarioName == dummyScenario {
```

Comment:
It is not obvious to see that this is related to https://github.com/DataDog/test-infra-definitions/pull/60/files/5e6e05a2f9d76c0d142b9f36925694a291a4b7c0#diff-dd2c0eb6ea5cfc6c4bd4eac30934e2d5746747af48fef6da689e85b752f39557R82. It would be great to add a comment I think.